src/dashboard/Settings/reducer.js

```diff
@@ -0,0 +1,92 @@
+import {
+  STORE_INCIDENTS_TYPE_SUCCESS,
+  SELECT_INCIDENT_TYPE,
+  GET_INCIDENTS_TYPE,
+  ADD_INCIDENT_TYPE,
+  UPDATE_INCIDENT_TYPE,
+  SELECT_COLOR_AUTOFILL,
+  FETCH_INCIDENT_TYPE_FAILURE,
+} from './actions';
+
+/**
+ * Incident reducers
+ *
+ * @function
+ * @name IncidentsType
+ *
+ * @param {Object} state - Initial state
+ * @param {Object} action - Redux action object
+ * @param {string} action.type - Action type
+ *
+ * @version 0.1.0
+ * @since 0.1.0
+ * */
+
+const initialState = {
+  filters: [],
+  data: [],
+  total: 0,
+  isLoading: false,
+  error: null,
+};
+
+export default function incidentsTypeSettings(state = initialState, action) {
```

Comment:
I think in the store there should be one store for incidentTypes

Reply:
what do you mean?

Reply:
Sorry I meant this should be renamed... to incidentTypes